src/components/menu-bar/menu-bar.jsx

```diff
@@ -28,7 +28,7 @@ import LoginDropdown from './login-dropdown.jsx';
 import SB3Downloader from '../../containers/sb3-downloader.jsx';
 import DeletionRestorer from '../../containers/deletion-restorer.jsx';
 import TurboMode from '../../containers/turbo-mode.jsx';
-import ConfirmReplaceHOC from './confirm-replace-hoc.jsx';
+import MenuBarHOC from '../../containers/menu-bar-hoc.jsx';
```

Comment:
I know that putting `MenuBarHOC` in the `containers/` directory doesn't match where other HOCs are -- but I think this should probably just be a standard container, rather than a general HOC. I didn't want to fully transform it into a container now, because I wanted to keep the scope of the changes narrow.